builtin/providers/google/resource_compute_address.go

```diff
@@ -14,7 +14,12 @@ func resourceComputeAddress() *schema.Resource {
 		Create: resourceComputeAddressCreate,
 		Read:   resourceComputeAddressRead,
 		Delete: resourceComputeAddressDelete,
-
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Any reason to do a custom implementation instead of [`schema.ImportStatePassthrough`](https://godoc.org/github.com/hashicorp/terraform/helper/schema#ImportStatePassthrough)?